hazelcast/util.py

```diff
@@ -66,6 +66,14 @@ def current_time():
     return time.time()
 
 
+def current_time_in_millis():
+    """
+    Returns the current time of the system in millis.
+    :return: (int), current time of the system in millis.
+    """
+    return int(time.time() * 1000)
```

Comment:
can you use a constant specifier instead of number 1000 here so that it is more understandable? E.g. MILLISECONDS_IN_A_SECOND